webcompat/templates/home-page/issue-wizard-form.html

```diff
@@ -21,14 +21,14 @@ <h3 class="page-heading">Filing a web compatibility issue</h3>
           <div class="form-text form-element js-Form-group form-label-message with-validation-icons">
             {{ form.url.label(class_='form-label is-hidden') }}
             {% if form.url.errors %}
-              {% for error in form.url.errors %}
```

Comment:
Plase make sure you follow the same indentation paradigm across the file. This PR introduces a lot of indentation changes that are not really required nor correct.

Reply:
This also has other occurences